comit/src/asset/ethereum/erc20.rs

```diff
@@ -100,6 +100,12 @@ impl From<Erc20Quantity> for blockchain_contracts::ethereum::TokenQuantity {
     }
 }
 
+impl From<BigUint> for Erc20Quantity {
```

Comment:
NACK. There is a `TryFrom` conversion that you can use which maintains the MAX_VALUE invariant.